src/Core/Style.js

```diff
@@ -820,6 +826,8 @@ class Style {
             style.point.opacity = opacity;
             style.point.radius = readVectorProperty(layer.paint['circle-radius']);
         } else if (layer.type === 'symbol') {
+            // if symbol we shouldn't draw stroke but defaut value is 1.
+            style.stroke.width = 0.0;
```

Comment:
no need for this comment I think, the code is clear enough

Reply:
A bit to much commentary is too bad ?
I can removed it, I wanted to be sure as up to now the stroke (of a polygon or line) were draw even with a width = 0.